navigation.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>CascadeUI | Navigation</title>
+</head>
+
+<body>
+
+    <div class="Header">
+        <div class="brand-name">
+            <a href="index.html">
+                CascadeUI
+            </a>
+        </div>
+
+        <div class="documentation-link">
+            <a href="docs.html">Documentation</a>
+        </div>
+
+    </div>
+
+
+    <div class="page-wrapper">
+        <div class="sidebar-nav">
+            <div class="heading">
+                <h3>get started</h3>
+                <ul>
+                    <li><a href="docs.html">Installation</a> </li>
+                </ul>
+            </div>
+            <div class="heading">
+                <h3>Components</h3>
+                <ul>
+                    <li><a href="alert.html">Alerts</a></li>
+                    <li><a href="button.html">Button</a></li>
+                    <li><a href="avatar.html">Avatar</a></li>
+                    <li><a href="badge.html">Badge</a></li>
+                    <li><a href="card.html">Card</a></li>
+                    <li><a href="image.html">Image</a></li>
+                    <li><a href="input.html">Input</a></li>
+                    <li><a href="grid.html">Grid</a></li>
+                    <li><a href="navigation.html">Navigation</a></li>
+                    <li><a href="rating.html">Rating</a></li>
+
+                </ul>
+            </div>
+
+        </div>
+
+
+
+        <div class="page-content">
+
+            <section id="navigation">
+
+                <h2>Navigation</h2>
+                <p>Navigation is simply a collection of buttons or links that navigate to that page or content.</p>
+                <h3>Navigation Bar(Simple Desktop)</h3>
+                <div class="navigation-container">
+                <div class="navigation-container-navbar">
+                    <div class="navigation">
```

Comment:
Instead of using div for every element, you can have a Nav tag here.  (semantic HTML)